src/app/services/search.service.ts

```diff
@@ -1,16 +1,23 @@
 import { SearchResult } from './../models/searchResult';
 import { Injectable } from '@angular/core';
 import { HttpClient } from '@angular/common/http';
-import { Observable } from 'rxjs';
+import { Observable, BehaviorSubject } from 'rxjs';
 
 @Injectable({
   providedIn: 'root'
 })
 export class SearchService {
-  baseURL = 'https://cors-anywhere.herokuapp.com/http://api.deezer.com/search?';
-  params = 'limit=2&';
+  private baseURL = 'https://cors-anywhere.herokuapp.com/http://api.deezer.com/search?';
```

Comment:
Move this to `environment.ts`

Reply:
This is one of the questions I wanted to ask when we gonna meet. About **.env** and system variables for angular.

Reply:
I wanna move it when proxy server will be up.